### ISSUE TITLE
flutter-3.1.0-9.0pre: compile error

### ISSUE THREAD

**Issue by xErik** [daohoangson/flutter_widget_from_html#800]
This is more of a heads up, probably not important for many developers at the moment. 

(I am forced to use `flutter 3.1.0` to get my App pass Google Play Store pre-launch testing.)

## Steps to Reproduce

`flutter_widget_from_html_core`
`flutter-3.1.0-9.0pre`

Compilation fails:

```
: Error: The non-abstract class 'FwfhTextStyle' is missing implementations for these members:
 - TextStyle.fontVariations
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.
../…/lib/fwfh_text_style.dart:11

class FwfhTextStyle extends _TextStyleProxy {
      ^^^^^^^^^^^^^
: Context: 'TextStyle.fontVariations' is defined here.
  final List<ui.FontVariation>? fontVariations;
../…/painting/text_style.dart:789

                                ^^^^^^^^^^^^^^
: Error: The method 'FwfhTextStyle.apply' has fewer named arguments than those of overridden method 'TextStyle.apply'.
  TextStyle apply({
../…/lib/fwfh_text_style.dart:43
```

<details>
  <summary>Testing environment</summary>

[√] Flutter (Channel beta, 3.1.0-9.0.pre, on Microsoft Windows [Version 10.0.19044.1766], locale
    en-DE)
    • Flutter version 3.1.0-9.0.pre at C:\Users\Erik\flutter_windows_3.0.2-stable\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f28e570c8c (4 weeks ago), 2022-06-14 13:39:33 -0500
    • Engine revision 74ee6b5afd
    • Dart version 2.18.0 (build 2.18.0-165.1.beta)
    • DevTools version 2.14.0

[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at C:\Users\Erik\AppData\Local\Android\sdk
    • Platform android-32, build-tools 32.1.0-rc1
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)   
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Android Studio (version 2021.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)

[√] VS Code (version 1.69.1)
    • VS Code at C:\Users\Erik\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.44.0

[√] Connected device (2 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 103.0.5060.114
    • Edge (web)   • edge   • web-javascript • Microsoft Edge 103.0.1264.49
    ! Device 3087SH3001018491 is offline.

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```
```
</details>

## Expected results

Co compile error.

## Actual results

Compile error.

**Comment by letungcntt** [comment]
Same this problem, how long to fix it ?

**Comment by netgomail** [comment]
I have the same problem Flutter version 3.1.0-0.0.pre.1869 on channel master

**Comment by netgomail** [comment]
> Same this problem, how long to fix it ?

add fontVariations from original text_style.dart for work

**Comment by kafumi** [comment]
#756 should resolve the issue. No new stable version including the fix is released yet but prerelease version `2.13.0-master.1` with the fix is available.
https://pub.dev/packages/fwfh_text_style/versions/2.13.0-master.1/changelog

**Comment by daohoangson** [comment]
A new version has been made available at https://pub.dev/packages/fwfh_text_style/versions/2.13.0. It should fix your issue with Flutter beta. Please try upgrading and let me know if it works for you.
